build.properties

```diff
@@ -13,4 +13,4 @@ tinker_version=2.10.0.+
 dynamic_tree_version=0.7.+
 jei_version=4.8.5.147
 c_b_version=14.0.128
-structurize_version=0.10.47-ALPHA
+structurize_version=0.10.58-ALPHA
```

Comment:
0.10.59 is the latest now, also you uploaded libs folder

Reply:
done